src/utils/sdfg_meta_dict.json

```diff
@@ -1 +1,7274 @@
+{
+  "__reverse_type_lookup__": {
+    "typeclass": {
+      "metatype": "typeclass",
+      "desc": "",
+      "category": "General",
+      "default": "int32",
+      "base_types": [
+        "bool",
+        "bool_",
+        "int8",
+        "int16",
+        "int32",
+        "int64",
+        "uint8",
+        "uint16",
+        "uint32",
+        "uint64",
+        "float16",
+        "float32",
+        "float64",
+        "complex64",
+        "complex128"
+      ],
+      "compound_types": {
+        "vector": {
+          "elements": { "type": "int", "default": 0 },
+          "dtype": { "type": "typeclass", "default": "bool" }
+        },
+        "pointer": { "dtype": { "type": "typeclass", "default": "bool" } },
+        "opaque": { "dtype": { "type": "typeclass", "default": "bool" } },
```

Comment:
opaque should have a string! something with the metadata is wrong

Reply:
Fixed.